explorer/src/components/account/TokenHistoryCard.tsx

```diff
@@ -100,9 +126,24 @@ function TokenHistoryTable({ tokens }: { tokens: TokenInfoWithPubkey[] }) {
     return history?.data !== undefined;
   });
 
+  const filteredFoundOldest = React.useMemo(() => {
```

Comment:
I don't think we need to compute both `allFoundOldest` and `filteredFoundOldest`. How about filtering tokens early on? I think it would clean up a lot of the code

Reply:
Okay, I'll look at doing that. I introduced filteredFoundOldest based on one of your other comments, but I agree we can probably filter them earlier in the process.

Reply:
Thanks for the suggestion, I think I just needed to see this from a different perspective.